selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AskForValueDialog.java

```diff
@@ -137,7 +138,10 @@ public void clickCancelButtonJava() {
   }
 
   public void typeAndWaitText(String text) {
-    input.sendKeys(text);
+    waitFormToOpen();
+    new WebDriverWait(seleniumWebDriver, TestTimeoutsConstants.LOAD_PAGE_TIMEOUT_SEC)
```

Comment:
Good catch!
Lets extract `new WebDriverWait(seleniumWebDriver, TestTimeoutsConstants.LOAD_PAGE_TIMEOUT_SEC)` into the separate constant and add similar verification in other methods.